modules/agent-install-configuring-for-disconnected-registry.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc
+
+:_content-type: Procedure[id="agent-install-configuring-for-disconnected-registry_{context}"]
+= Configuring the Agent-based Installer to use mirrored images
+
+You must use the output of either the `oc adm release mirror` command or the oc-mirror plugin to configure the Agent-based Installer to use mirrored images.
+
+.Procedure
+
+. If you used the oc-mirror plugin to mirror your release images:
+
+.. Open the `imageContentSourcePolicy.yaml` located in the results directory, for example `oc-mirror-workspace/results-1682697932/`.
+
+.. Copy the text in the `repositoryDigestMirrors` section of the yaml file.
+
+. If you used the `oc adm release mirror` command to mirror your release images:
+
+* Copy the text in the `imageContentSources` section of the command output.
+
+. Paste the copied text into the `imageContentSources` field of the `install-config.yaml` file.
+
+. Add the certificate file used for the mirror registry to the `additionalTrustBundle` field of the yaml file.
++
+[IMPORTANT]
+====
+The value must be the contents of the certificate file that you used for your mirror registry.
+The certificate file can be an existing, trusted certificate authority, or the self-signed certificate that you generated for the mirror registry.
+====
++
+.Example `install-config.yaml` file
+
+[source,yaml]
+----
+  additionalTrustBundle: |
+    -----BEGIN CERTIFICATE-----
+    ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+    -----END CERTIFICATE-----
+----
+
+. If you are using {ztp} manifests: add the `registries.conf` and `ca-bundle.crt` files  to the `mirror` path to add the mirror configuration in the agent ISO image.
++
+[NOTE]
+====
+You can create the `registries.conf` file from the output of either the `oc adm release mirror` or `oc mirror` command. The format of the `/etc/containers/registries.conf` file has changed . It is now version 2 and in TOML format.
+====
++
+.Example `registries.conf` file
+
+[source,toml]
+----
+[[registry]]
+location = "registry.ci.openshift.org/ocp/release" mirror-by-digest-only = true
+
+[[registry.mirror]] location = "virthost.ostest.test.metalkube.org:5000/localimages/local-release-image"
+
+[[registry]]
+location = "quay.io/openshift-release-dev/ocp-v4.0- art-dev" mirror-by-digest-only = true
```

Comment:
Shouldn't have apace in 'ocp-v4.0- art-dev'

Reply:
fixed, thanks for catching this! Let me know if you have any other feedback